main/btrfs-progs/APKBUILD

```diff
@@ -2,23 +2,23 @@
 # Contributor: Łukasz Jendrysik <scadu@yandex.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=btrfs-progs
-pkgver=4.16
-pkgrel=1
+pkgver=4.16.1
+pkgrel=0
 pkgdesc="BTRFS filesystem utilities"
 url="https://btrfs.wiki.kernel.org"
 arch="all"
 license="GPL-2.0-or-later LGPL-3.0-or-later"
 depends=""
 depends_dev="linux-headers"
-makedepends="$depends_dev acl-dev asciidoc attr-dev autoconf automake bash e2fsprogs-dev
+makedepends="$depends_dev acl-dev asciidoc attr-dev autoconf automake e2fsprogs-dev
 	libtool lzo-dev py3-setuptools python3-dev util-linux-dev xmlto zlib-dev"
 subpackages="$pkgname-doc $pkgname-dev $pkgname-bash-completion:bashcomp:noarch
 	py-$pkgname:py $pkgname-libs $pkgname-extra"
 source="https://www.kernel.org/pub/linux/kernel/people/kdave/$pkgname/$pkgname-v$pkgver.tar.xz
 	includes.patch
 	musl-compat.patch
 	"
-builddir="$srcdir/$pkgname-v$pkgver"
+builddir="$srcdir"/$pkgname-v$pkgver
```

Comment:
Why this change? The current form is preferred.

Reply:
@jirutka Noted. Will amend in future commits.